.ci/azure-wheel-helpers/README.md

```diff
@@ -0,0 +1,118 @@
+## Azure Wheel Helpers
+
+This repository holds a collection of wheel helpers designed by the [Scikit-HEP][] project to build Python Wheels on [Azure DevOps][]. This is designed for packages that require building; if you have a pure-Python project, producing a universal wheel is trivial without this helper collection. This collection assumes some standard paths and procedures, though *some* of them can be customized.
+
+Azure provides manual pipeline triggering and release pipelines, making it slighly better suited for this than GitHub Actions, though otherwise they are *very* similar.
+
+### Supported platforms and caveats
+
+TLDR: Python 2.7, 3.6, 3.7, and 3.8  on all platforms, along with 3.5 on Linux.
```

Comment:
Should we just drop 3.5 altogether?

Reply:
We probably should soon, but I'd have to filter it out of the manylinux versions (like I filter 3.4 already). I'm hoping to later add a nice way to do this per project.

Reply:
Anything in azure-wheel-helpers should not be changed, and can be updated from the main repository with a `git subtree pull <options>`.